### ISSUE TITLE
the latest data for cn is not updated

### ISSUE THREAD

**Issue by wanpengxyzz** [microsoft/qlib#1490]
## To Reproduce

1.python scripts/get_data.py qlib_data --target_dir ~/qlib_data/cn_data --region cn
can not get the lastest data。

## Environment

**Note**: User could run `cd scripts && python collect_info.py all` under project directory to get system information
and paste them here directly.

 - Qlib version:
 - Python version:
 - OS (`Windows`, `Linux`, `MacOS`):
 - Commit number (optional, please provide it if you are using the dev version):

**Comment by Fivele-Li** [comment]
Hi, could you please send us more details about your error(e.g the backpack information, the folder structure of your data)?

**Comment by yifei-zhhhh** [comment]
Same issue here. I think the data is only updated to 2020/09/25 as I tried.

**Comment by SunsetWolf** [comment]
You are right, use the command: `python scripts/get_data.py qlib_data --target_dir ~/qlib_data/cn_data --region cn` downloaded the data, the latest date of this data is 2020-09-25, is because this data is pre-prepared data that is then stored in the cloud for you to download and use.
If you need new data, you can prepare the data yourself, then standardize the data and dump it into qlib format. qlib also provides [scripts](https://github.com/microsoft/qlib/tree/main/scripts/data_collector/yahoo) for you to use.
